### PR TITLE
Refine typo of english document of paddle.prod

### PR DESCRIPTION
**Pull request by gfwm2013**
### PR types
Others

### PR changes
Docs

### Describe
Refine the english document of paddle.prod

### CODE REVIEW

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by wangchaochaohu — DISMISSED** [review]

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -1639,7 +1639,7 @@ def prod(x, axis=None, keepdim=False, dtype=None, name=None):
     Compute the product of tensor elements over the given axis.
 
     Args:
-        x(Tensor): An N-D Tensor, the data type is float32, float64, int32 or int64.
+        x(Tensor): The input tensor, it's data type should be float32, float64, int32, int64.
```
Comment (zhhsplendid) [inline python/paddle/tensor/math.py]: Grammar: "it's data" -> "its data"

"it's" means "it is" or "it has"

Comment (gfwm2013) [reply python/paddle/tensor/math.py]: Thanks, done.

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**